gwpy/timeseries/timeseries.py

```diff
@@ -986,7 +990,15 @@ def zpk(self, zeros, poles, gain, analog=True, **kwargs):
 
         >>> data2 = data.zpk([100]*5, [1]*5, 1e-10)
         """
-        return self.filter(zeros, poles, gain, analog=analog, **kwargs)
+
```

Comment:
Can you please remove the blank line between the docstring end and the code? There are a few examples in the test modules as well, so please remove those if you can.

```suggestion

```